airports.json

```diff
@@ -2720,8 +2720,8 @@
         "country": "US",
         "elevation": 89,
         "lat": 29.9409722222,
-        "lon": -96.2452500000,
-        "tz": "America/Chicago"
+        "lon": -96.24525,
```

Comment:
Not sure the trailing `0`'s should be removed 🤔

Reply:
As mentioned, it's programmatic and a byproduct of using python 3.8 json.dump() https://docs.python.org/3/library/json.html#json.dump

It is arguably mathematically incorrect to have them there to begin with, so unless someone is going to manually re-enter all the zeros, it's a byproduct of getting a vastly upgraded database!